run.py

```diff
@@ -85,14 +85,15 @@ def run(full_text_links, **kwargs):
                 i+1, total, record['bibcode'])
         )
 
-        if max_queue_size and i > max_queue_size:
+        if max_queue_size and i >= max_queue_size:
             logger.info('Max_queue_size reached, stopping...')
             break
 
         if diagnose:
             print("[{}/{}] Calling 'task_check_if_extract' with '{}'".format(i+1, total, str(record)))
         logger.debug("[%i/%i] Calling 'task_check_if_extract' with '%s'", i+1, total, str(record))
-        tasks.task_check_if_extract.delay(record)
+        #tasks.task_check_if_extract.delay(record)
+        tasks.task_check_if_extract(record) # Treat synchronously to avoid saturating NFS mount access
```

Comment:
In adswhy right now is more efficient when we do the check extract sequentially (it requires plenty of read access to a NFS mounted volume) rather than in parallel (tested with different number of workers), maybe in the new machine this conclusion will change but so far it is better this way.